content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,110 @@
+
+When the status indicator displays **Watching for filesystem changes**, your files are available to the VM through all the standard bind mount mechanisms, whether that's `-v` in the command line or specified in your `compose.yml` file.
+
+>**Note**
+>
+> When you create a new service, setting the [bind mount option consistency](../engine/reference/commandline/secret_create.md#options-for-bind-mounts) to `:consistent` bypasses synchronized file sharing. 
+
+## Explore your file share instance
+
+The **Synchronized file shares** section displays all your file share instances and provides useful information about each instance including:
+- The origin of the file share content
+- A status update
+- How much space each file share is using
+- The number of filesystem entry counts
+- The number of symbolic links
+- Which container(s) is using the file share instance
+
+Selecting a file share instance expands the dropdown and exposes this information.
+
+## Use `.syncignore`
+
+You can use a `.syncignore` file at the root of each file share, to exclude local files from your file share instance. It supports the same syntax as `.dockerignore` files and excludes, and/or re-includes, paths from synchronization. `.syncignore` files are ignored at any location other than the root of the file share.
+ 
+Some example of things you might want to add to your `.syncignore` file are:
+- Large dependency directories, for example `node_modules` and `composer` directories (unless you rely on accessing them via a bind mount)
+- `.git directories` (again, unless you need them)
+
+In general, use your `.syncignore` file to exclude items that aren't critical to your workflow, especially those that would be slow to sync or use significant storage.
+
+## Frequently asked questions
+
+### What is the difference between Synchronized file sharing and the Mutagen extension?
+
+Since Docker [acquired Mutagen](https://www.docker.com/blog/mutagen-acquisition/), Synchronized file sharing is essentially the direct integration of the Mutagen extension into Docker Desktop. However, Synchronized file sharing provides an improved user experience and the ability to use a [`.syncignore` file](#use-syncignore). You also no longer have to set the default ownership on files as this is now handled automatically.
+
+## Known issues
+
+- Upon launching Docker Desktop, it can take between 5-10 seconds for Synchronized file sharing to fully initialize. During this window, file share instances display as **Connecting** and any new containers created during this window won't replace bind mounts with Synchronized file sharing.
+
+- `.syncignore` changes won't cause deletion until file share recreation. In other words, files that become ignored due to a modification to `.syncignore` are left in place, but no longer update through synchronization.
```

Comment:
"..changes won't cause deletion.." Do we mean the files are not deleted? Wondering if we can rephrase it

Reply:
nit: Should we hyphenate re-creation for clarity